### DISCUSSION TITLE
How do i launch neovide inside a terminal instead of it opening in another window

### DISCUSSION THREAD

**Discussion by chihabhajji** [neovide/neovide#1104] [Q&A] [answered]
i'm using windows

**Comment by LoipesMas** [comment, marked as answer]
Neovide isn't a terminal application, so you can't run it in a terminal.

**Reply by dineshKumar777** [reply]
@LoipesMas  is it possible to open inside same neovide instance, instead of opening separate??

**Reply by LoipesMas** [reply]
@dineshKumar777 you mean opening neovide inside neovide? or opening a new file inside already open neovide instance?

**Reply by dineshKumar777** [reply]
Opening a new file inside already open neovide instance from nvim terminal split.
in image, neovide "filename" will open file in new neovide instance. Is it possible to open in same neovide instance???

https://github.com/nikvdp/neomux - like in this project.

![image](https://user-images.githubusercontent.com/21097695/147243482-f3cf56a2-9d9e-46b3-b92b-7d3003da09d0.png)

**Reply by LoipesMas** [reply]
You can use [neovim-remote](https://github.com/mhinz/neovim-remote). Just replace `nvim` with `neovide`

**Reply by dineshKumar777** [reply]
I tried this project long back.. i couldn't able to make it work both neovim-remote and neomux properly. will give it a try this weekend again.

Small doubt, neovide already communicating with external nvim process right.. is it possible to achieve this without configuring additional plugin??
